Libraries/JavaScriptAppEngine/Initialization/InitializeJavaScriptAppEngine.js

```diff
@@ -25,11 +25,11 @@
 require('regenerator/runtime');
 
 if (typeof GLOBAL === 'undefined') {
-  GLOBAL = this;
+  global.GLOBAL = this;
```

Comment:
Should we get rid of the `GLOBAL` global variable and just use the node standard `global` everywhere? Except for this file it is used in 1 or 2 other places in the codebase.